svthalia/concrexit
# Move partner events to events package as 'external events'

Closes #1892 

### Summary
This moves the idea of external events from the partner package to events. They are no longer connected to a partner directly and can now be used by everyone to show the exact event. Think of the Beestfeest or SNiC.

### How to test
Steps to test the changes you made:
1. Go to the events pages and check you can edit external events
2. Check that the external events are shown in the calendar just like partner events were before
3. Check that the external events API works with event permissions
4. Check that old partner events are properly migrated
